app/_plugins/filters/to_badges.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module BadgeFilter
+  def to_badges(input)
```

Comment:
I don't think this is used anymore

Reply:
If you make a change per below, can you remove this as well?  Thank you!